k8s/service.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: opensergo-control-plane
+  namespace: opensergo-system
+spec:
+  ports:
+    - port: 10246
+      protocol: TCP
+      targetPort: 10246
```

Comment:
`tartgetPort` can be given `grpc` which was defined in `Deployment`, use `port number` directly is not convenient

Reply:
done